openapi.yml

```diff
@@ -516,12 +566,6 @@ components:
           $ref: "#/components/schemas/SigningScheme"
         signature:
           $ref: "#/components/schemas/Signature"
-        feePolicies:
```

Comment:
@MartinquaXD I think this was incorrect from https://github.com/gnosis/solvers/pull/8

Reply:
Why is that? With the introduction of the `fee-handler` parameter these policies can now be forwarded to the solver, no?

Reply:
Yes, but this is the `JitOrder`. The `feePolicies` are forwarded only for `Orders`, as I added [here](https://github.com/gnosis/solvers/pull/18/files#diff-5016bdc926b6c0f649f363739d09159a1db664aae19a08fa6b38b03406f040dcR217), or maybe am I missing something? :thinking:

Reply:
Oh, didn't notice it was on the JIT order. Then it's indeed a mistake. 👍